schema/codelists/contractStatus.csv

```diff
@@ -1,7 +1,5 @@
-Code,Title,Description
-pending,Pending,"This contract has been proposed, but is not yet in force. It might be awaiting signature."
-active,Active,"This contract has been concluded by all the parties, and is now legally in force."
-cancelled,Cancelled,This contract has been cancelled prior to being concluded.
-terminated,Terminated,"This contract was concluded and in force, and has now come to a close. This might be due to the successful completion of the contract, or might be early termination due to some non-completion."
-terminatedEarly,Terminated early,"This contract was concluded and in force, and is terminated early due to some non-completion."
-terminatedSuccessfully,Terminated successfully,"This contract was concluded and in force, and is terminated due to its successful completion."
+Code,Title,Description,Deprecated,Deprecation note
+pending,Pending,"The award was made and the standstill period, if any, has ended. The contract is not yet concluded (for example, not signed by all parties).",1.2,Deprecated because the same information can be determined by the absence of `contracts.dateSigned`.
+active,Active,"The contract was concluded (for example, signed by all parties). The contract is not yet terminated.",1.2,Deprecated because the same information can be determined by comparing `contracts.period` to `date`.
```

Comment:
```suggestion
active,Active,"The contract was concluded (for example, signed by all parties). The contract is not yet terminated.",1.2,Deprecated because the same information can be determined by the presence of `contracts.dateSigned` and by comparing `contracts.period` to `date`.
```